### PR TITLE
[EDU-5020] fix: adjust time-related metrics in Observe Docs

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-5020] 

### Changes

Adjust time-related metrics in Observe Docs:

- Real-Time Events GraphQL API Fields
- Real-Time Events Reference.
- Real-Time Metrics GraphQL API Fields
- Data Stream Reference

### Additional links

n/a.

[EDU-5020]: https://aziontech.atlassian.net/browse/EDU-5020?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx [hunk 1]
```diff
@@ -135,11 +135,11 @@ Veja cada campo disponível e suas descrições abaixo.
 | upstreamBytesSent | Número de bytes enviados para a origem. Exemplo: `2733` |
 | upstreamBytesSentStr | Lista com todas as respostas `upstreamBytesSent`. |
 | upstreamCacheStatus | Status do cache local do edge. Pode ser: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT` ou `-` |
-| upstreamConnectTime | Tempo que o edge leva para estabelecer uma conexão com a origem em milissegundos. No caso de TLS, inclui o tempo gasto no handshake. Exemplo: `0.123`. **`0` no caso do KeepAlive e `-` no caso de cache.** |
+| upstreamConnectTime | Time it takes for the edge to establish a connection with the origin, in seconds. In the case of TLS, it includes time spent on handshake. Example: `0.123`. **Returns `0` for KeepAlive and `-` for cache** |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:138]: @MarianaAguilera shouldn't this be in portuguese?

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx [hunk 2]
```diff
@@ -117,15 +117,15 @@ Quando um campo é o resultado de algum tipo de cálculo, como uma soma, ele é
 | remoteAddressClass | Classe do endereço de IP da origem que gerou a requisição. Exemplo: `44.192.0.0/11` |
 | requestLength | Tamanho da requisição em bytes, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é resultado de uma soma. Exemplo: `167` |
 | requestMethod | Método da requisição. Exemplo: `GET` ou `POST`. |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente com resolução de milissegundos. Este campo é resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx:120]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx [hunk 3]
```diff
@@ -176,14 +176,14 @@ Quando um campo é o resultado de algum tipo de cálculo, como uma soma, ele é
 | host | Informação de host enviada na linha da requisição. Armazena: nome do host da linha da requisição, ou o nome do host do campo Host do campo host do cabeçalho, ou o nome do servidor correspondente à requisição. Exemplo: `g1sdetynmxe0ao.map.azionedge.net` |
 | remoteAddressClass | Classe do endereço de IP da origem que gerou a requisição. Exemplo: `44.192.0.0/11` |
 | requestMethod | Método da requisição. Exemplo: `GET` ou `POST`. |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente com resolução de milissegundos. Este campo é resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx:179]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx [hunk 4]
```diff
@@ -251,15 +251,15 @@ Os seguintes conjuntos de dados foram descontinuados. Recomenda-se utilizar os [
 | remoteAddressClass | Classe do endereço de IP da origem que gerou a requisição. Exemplo: `44.192.0.0/11` |
 | requestLength | Tamanho da requisição em bytes, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é resultado de uma soma. Exemplo: `167` |
 | requestMethod | Método da requisição. Exemplo: `GET` ou `POST`. |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente com resolução de milissegundos. Este campo é resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/metrics-campos.mdx:254]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 5]
```diff
@@ -87,7 +87,7 @@ Exibe os registros de eventos de requisições feitas para suas [edge applicatio
 | Proxy Status | Código de status de erro HTTP ou da origem quando nenhuma resposta é obtida da origem. Exemplo: `520`. **Em caso de cache, a resposta é `-`.** |
 | Request Length | Tamanho da requisição, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `167` |
 | Request Method | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| Request Time | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:90]:
```suggestion
| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 6]
```diff
@@ -176,7 +176,7 @@ Exibe os registros de eventos de requisições feitas às edge applications que
 | Remote Addr | Endereço IP da origem que gerou a requisição. Exemplo: `127.0.0.1` |
 | Remote Port | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
 | Request Method | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| Request Time | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:179]:
```suggestion
| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 7]
```diff
@@ -213,7 +213,7 @@ Exibe os registros de eventos de requisições feitas às edge applications que
 | Remote Port | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
 | Request Length | Tamanho da requisição, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `167` |
 | Request Method | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| Request Time | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:216]:
```suggestion
| Request Time | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx [hunk 8]
```diff
@@ -311,7 +311,7 @@ Os seguintes conjuntos de dados foram descontinuados. Recomenda-se utilizar os [
 | remotePort | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
 | requestLength | Tamanho da requisição, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `167` |
 | requestMethod | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:314]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx [hunk 9]
```diff
@@ -220,7 +220,7 @@ Veja cada campo disponível e suas descrições abaixo.
...
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:223]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx [hunk 10]
```diff
@@ -182,7 +182,7 @@ Veja cada campo disponível e suas descrições abaixo.
 | remoteAddr | Endereço IP da origem que gerou a requisição. Exemplo: `127.0.0.1` |
 | remotePort | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
 | requestMethod | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:185]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx [hunk 11]
```diff
@@ -106,7 +106,7 @@ Veja cada campo disponível e suas descrições abaixo.
 | requestId | Identificador exclusivo da requisição. Exemplo: `5f222ae5938482c32a822dbf15e19f0f` |
 | requestLength | Tamanho da requisição, incluindo a linha da requisição, cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `167` |
 | requestMethod | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
-| requestTime | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: Exemplo: `0.234` |
```
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:109]:
```suggestion
| requestTime | Tempo de processamento da requisição, em segundos, desde que os primeiros bytes foram lidos a partir do cliente. Este campo é resultado de uma soma. Exemplo: `0.234` |
```

**Review by guiafonso-ol — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/events-campos.mdx:138]:
```suggestion
| upstreamConnectTime | Tempo que leva para o edge estabelecer uma conexão com a origem, em segundos. No caso de TLS, inclui o tempo gasto no handshake. Exemplo: `0.123`. **Retorna `0` para KeepAlive e `-` para cache** |
```

**Pull request merged**